packages/firestore/test/unit/remote/node/serializer.test.ts

```diff
@@ -200,16 +204,13 @@ describe('Serializer', () => {
       ];
 
       for (let i = 0; i < examples.length; i++) {
-        const example = examples[i];
-        const obj = s.toValue(
-          new fieldValue.TimestampValue(Timestamp.fromDate(example))
-        );
-        const expected = { timestampValue: expectedJson[i] };
-        expect(obj).to.deep.equal(expected);
-        expectValue(obj, 'timestampValue').to.deep.equal(
-          new protos['google']['protobuf'].Timestamp(expectedJson[i]),
-          'for date ' + example
+        const value: fieldValue.FieldValue = new fieldValue.TimestampValue(
+          Timestamp.fromDate(examples[i])
         );
+        const proto: api.Value = {
+          timestampValue: (expectedJson[i] as unknown) as string
```

Comment:
I'm not sure why the proto's timestampValue is typed as a string? But I also have no idea how protobufjs works.

I also needed the same trick down below in the bytes section, though at least in that case, it sort of makes sense... since at least some of the time this is actually a base64 encoded string.